src/server/page_server.hpp

```diff
@@ -61,8 +58,7 @@ class page_server
     void finalize_log_page_fetcher ();
 
   private:
-    using active_tran_server_request_queue = cubcomm::request_sync_send_queue<active_tran_server_conn, std::string>;
-    using active_tran_server_request_autosend = cubcomm::request_queue_autosend<active_tran_server_request_queue>;
+    using ats_t = cubcomm::request_sync_client_server<ps_to_ats_request, ats_to_ps_request, std::string>;
```

Comment:
Same as "ps", better to have a verbose name. Mirror the name chosen for ps.

Reply:
Done